tools/ci_build/build.py

```diff
@@ -1210,6 +1213,12 @@ def run_adb_shell(cmd):
             adb_push('onnx_test_runner', device_dir, cwd=cwd)
             adb_shell('chmod +x {}/onnx_test_runner'.format(device_dir))
             run_adb_shell('{0}/onnxruntime_test_all'.format(device_dir))
+            if args.build_java:
+                adb_install(
+                    os.path.join(build_dir, "Debug", "java", "androidtest", "android", "app", "build", "outputs", "apk", "debug", "app-debug.apk"))
```

Comment:
we probably don't want to hardcode "Debug" here, or it won't work for other build configurations.
could use `get_config_build_dir(build_dir, config)` to get the start of the path.